packages/manager/apps/pci-load-balancer/package.json

```diff
@@ -31,9 +31,11 @@
     "@tanstack/react-query": "^5.50.1",
     "@tanstack/react-query-devtools": "^5.50.1",
     "@tanstack/react-table": "^8.17.3",
+    "axios": "^1.7.7",
```

Comment:
The only use of Axios exists on the PR is for `AxiosError` (correct me if am wrong) , and there are already objects available from the common internal module that you can use:

`import { ApiError } from '@ovh-ux/manager-core-api';`
Or ( depend to u're use case )
`import { ResponseAPIError } from '@ovh-ux/manager-pci-common';`